### ISSUE TITLE
It doesn't seem to be autocompiling... Or do I understand it wrong?

### ISSUE THREAD

**Issue by MathiasSM** [jrief/django-sass-processor#55]
I'm certainly new to Django, and I don't really know how it loads the apps declared on `settings.py`, but following the few steps on the README (that is, installing dependencies, adding it as an app on `settings.py`) and placing 

```python
STATIC_URL = '/static/'
STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
MEDIA_URL = '/media/'
MEDIA_ROOT = os.path.join(BASE_DIR, 'media/')
```

I got it to work... But it compiles only when I call the runserver command (or the compilescss command)... Shouldn't it watch the files to see if something changed and compile them at some point? Or did I misunderstand the process?

**Comment by jrief** [comment]
it only compiles during template rendering, if that containins a ``{% sass_src ... %}`` tag.

**Comment by Sheepzez** [comment]
Would it be possible to have an option to auto-recompile SASS files when a SASS file changes during development? Right now it seems the only option is to manually delete the CSS files, causing a recompilation.

**Comment by jrief** [comment]
This is the default, otherwise **django-sass-processor** would be pretty useless :)

Therefore please check, if the file is not recompiled, or if the browser loads it from its cache.

**Comment by AndreasBackx** [comment]
There however is a bug related to saving, see #32. Maybe it's somehow related.

**Comment by jrief** [comment]
closed via #32
